met-web/src/components/survey/building/index.tsx

```diff
@@ -219,11 +220,38 @@ const SurveyFormBuilder = () => {
                             <Switch
                                 checked={formDefinition.display === 'wizard'}
                                 onChange={(e) => {
-                                    if (e.target.checked) {
-                                        setFormDefinition({ display: 'wizard', components: [] });
-                                        return;
-                                    }
-                                    setFormDefinition({ display: 'form', components: [] });
+                                    dispatch(
+                                        openNotificationModal({
+                                            open: true,
+                                            data: {
+                                                header: 'Change Survey Type',
+                                                subText: [
+                                                    {
+                                                        text: `You will be changing the survey type from ${
+                                                            formDefinition.display === 'wizard' ? 'multi-page' : 'form'
+                                                        } to ${
+                                                            formDefinition.display === 'wizard' ? 'form' : 'multi-page'
```

Comment:
from form to multi-page. there both forms right. Maybe from single page to multi page